<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvoiceWindow</class>
 <widget class="QWidget" name="InvoiceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Faktura</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="invoiceInfoGroupBox">
     <property name="title">
      <string>Podaci o fakturi</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="invoiceNumberLabel">
        <property name="text">
         <string>Broj fakture:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="invoiceNumberLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="clientLabel">
        <property name="text">
         <string>Klijent:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QComboBox" name="clientComboBox"/>
        </item>
        <item>
        <widget class="QPushButton" name="addClientButton">
        <property name="text">
        <string>+</string>
        </property>
        <property name="maximumWidth">
        <number>30</number>
        </property>
        </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Opis / relacija:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="descriptionLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="dateLabel">
        <property name="text">
         <string>Datum:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="itemsTableWidget">
     <property name="columnCount">
      <number>5</number>
     </property>
     <column>
      <property name="text">
       <string>Roba / Usluga</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Koliƒçina</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Jedinica</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cijena</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Iznos</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="addItemButton">
       <property name="text">
        <string>+ Dodaj stavku</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeItemButton">
       <property name="text">
        <string>- Ukloni stavku</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="totalLabel">
       <property name="text">
        <string>Ukupno:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="totalLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    <item>
    <widget class="QCheckBox" name="pdvCheckBox">
    <property name="text">
    <string>PDV 25%</string>
    </property>
    <property name="checked">
    <bool>true</bool>
    </property>
    </widget>
    </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Spremi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Zatvori</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
